tests/functional/test_srpm.py

```diff
@@ -57,6 +58,25 @@ def test_srpm_command(cwd_upstream_or_distgit):
     build_srpm(srpm_path)
 
 
+def test_action_output(upstream_and_remote):
+    upstream_repo_path = upstream_and_remote[0]
+    packit_yaml_path = upstream_repo_path / ".packit.json"
+    packit_yaml_dict = json.loads(packit_yaml_path.read_text())
+    # http://www.atlaspiv.cz/?page=detail&beer_id=4187
+    the_line_we_want = "MadCat - Imperial Stout Rum Barrel Aged 20°"
```

Comment:
I hope it's legal to quote it here...;)

Reply:
it's just a beer name :D the page above has the name as well

I can pick some boring line if you want